lms/lib/comment_client/thread.py

```diff
@@ -31,7 +32,7 @@ class Thread(models.Model):
         'endorsed', 'read'
     ]
 
-    initializable_fields = updatable_fields + ['thread_type']
+    initializable_fields = updatable_fields + ['thread_type', 'context']
```

Comment:
Nit: Not really your fault, but it would be nice to have comments that outline the differences in these types of fields. Also, maybe why `thread_type` appears twice in the list.